README.md

```diff
@@ -18,17 +18,16 @@ $ pip install https://github.com/benchopt/benchopt
 Usage
 -----
 
-The main entry point to run benchmark with benchOpt is the `benchopt` command line tool, based on `click`.
-To run a benchmark to options are possible.
-
-When the solvers are installed directly on the system, it is possible to use the `run` command:
+The `benchopt` command line tool is based on `click`. To run benchmarks on all problems, all datasets and with all solvers, run:
```

Comment:
This does not work anymore. I think it does not make sense to try and run all the benchmark as it is too slow... So I would say that you always need to specify at least one becnhmark.
WDYT?